modules/dev-utils/src/main/java/org/apache/ignite/development/utils/indexreader/IgniteIndexReader.java

```diff
@@ -491,6 +505,28 @@ public void readIdx() {
         }
     }
 
+    /**
+     * Print partitions reading exceptions.
+     *
+     * @param partStoresErrors partitions reading exceptions.
+     */
+    private void printFileReadingErrors(Map<Integer, List<Throwable>> partStoresErrors) {
+        List<Throwable> idxPartErrors = partStoresErrors.get(INDEX_PARTITION);
+
+        if (idxPartErrors != null) {
```

Comment:
It is better to use:
`org.apache.ignite.internal.util.lang.GridFunc#isEmpty(java.util.Map<?,?>)`